### PR TITLE
[SP] UX with polymer

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
First version to leverage Polymer in SP sample.

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@leonhsl @halton Please help to review. Thanks!

**Comment by halton** [comment]
@DonnaWuDongxia please fix the style check errors.

**Comment by DonnaWuDongxia** [comment]
@halton updated, thanks!

File: sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html [hunk 1]
```diff
@@ -1,18 +1,229 @@
-<link rel="import" href="../elements.html">
-
+<link rel="import" href="../toggle-image-button/toggle-image-button.html">
+<link rel="import" href="../switch-image-button/switch-image-button.html">
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html:2]: Why not import `elements.html`? We should keep only one sentence to import html.

File: sample/src_polymer/scripts/common/utils.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var RSUtils = {
```
Comment (leonhsl) [inline sample/src_polymer/scripts/common/utils.js:1]: Maybe we'd better place both `spMath.js` and `utils.js` to `src_polymer/common`, which is parallel to `src_polymer/libs` as before?

**Comment by leonhsl** [comment]
Other than the 2 comments above, lgtm.
Will merge now and you can address the comments with a follow up PR. Thanks~

**Comment by leonhsl** [comment]
Merged

File: sample/src_polymer/elements/scene-perception-pages/main.js [hunk 3]
```diff
@@ -0,0 +1,301 @@
+const MIN_ACCEPTABLE_QUALITY = 0.25;
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/main.js:1]: Maybe we'd better use a closure to hide all the variables/functions from `document`? Only export necessary ones.

**Pull request merged**